src/ucp/am/eager_multi.c

```diff
@@ -207,10 +208,14 @@ static UCS_F_ALWAYS_INLINE void ucp_proto_am_eager_multi_zcopy_add_footer(
 {
     size_t user_hdr_size = req->send.msg_proto.am.header_length;
     size_t footer_size, footer_offset;
+    ucp_am_first_ftr_t *first_ftr;
 
     if (req->send.state.dt_iter.offset == 0) {
         footer_size   = sizeof(ucp_am_first_ftr_t) + user_hdr_size;
         footer_offset = 0;
+        first_ftr     = UCS_PTR_BYTE_OFFSET(req->send.msg_proto.am.reg_desc + 1,
+                                            user_hdr_size);
+        ucp_am_fill_first_footer(first_ftr, req);
```

Comment:
can you please add comment that it also fills mid footer which is reused by all fragments because it the same